### ISSUE TITLE
Cannot find module '../.prettierrc'

### ISSUE THREAD

**Issue by ryicoh** [victorgarciaesgi/nuxt-typed-router#17]
not work for me...

```
ERROR  Cannot find module '../.prettierrc'
Require stack:
- /Users/ryicoh/workspace/school-client/node_modules/nuxt-typed-router/lib/save.js
- /Users/ryicoh/workspace/school-client/node_modules/nuxt-typed-router/lib/module.js
- /Users/ryicoh/workspace/school-client/node_modules/@nuxt/core/dist/core.js
- /Users/ryicoh/workspace/school-client/node_modules/@nuxt/cli/dist/cli-index.js
- /Users/ryicoh/workspace/school-client/node_modules/@nuxt/cli/dist/cli.js
- /Users/ryicoh/workspace/school-client/node_modules/@nuxt/typescript-runtime/bin/nuxt-ts.js
```

**Comment by victorgarciaesgi** [comment]
@ryicoh I though I fixed this bug sorry! Looking at it tomorrow

**Comment by ryicoh** [comment]
thanks 🙇‍♀️

**Comment by victorgarciaesgi** [comment]
@ryicoh Should be fixed in `0.2.14` !
